ethereum/spec/src/testFixtures/java/tech/pegasys/teku/spec/TestSpecFactory.java

```diff
@@ -492,19 +499,17 @@ private static SpecConfigAndParent<? extends SpecConfig> getFuluSpecConfig(
             }));
   }
 
-  public static Spec createMinimalWithCapellaDenebAndElectraForkEpoch(
-      final UInt64 capellaForkEpoch, final UInt64 denebForkEpoch, final UInt64 electraForkEpoch) {
+  public static Spec createMinimalWithCapellaDenebElectraAndFuluForkEpoch(
+      final UInt64 capellaForkEpoch,
+      final UInt64 denebForkEpoch,
+      final UInt64 electraForkEpoch,
+      final UInt64 fuluForkEpoch) {
     final SpecConfigAndParent<? extends SpecConfig> config =
-        getElectraSpecConfig(
-            Eth2Network.MINIMAL, capellaForkEpoch, denebForkEpoch, electraForkEpoch);
-    return create(config, SpecMilestone.ELECTRA);
+        getFuluSpecConfig(
+            Eth2Network.MINIMAL, capellaForkEpoch, denebForkEpoch, electraForkEpoch, fuluForkEpoch);
+    return create(config, SpecMilestone.FULU);
   }
 
```

Comment:
I'd restore removed javadoc here and change ELECTRA to FULU:
```java
  // Our current config files contain FULU params.
  // So all specConfigs created from them will be FULU.
  // Here we just want to make sure that a given config supports the given milestone
  // (which useless in theory because they are all FULU)

```